src/test_order.py

```diff
@@ -1,19 +1,59 @@
 import unittest
+import json
 
-from src.martins_script import Order
+from eth_account import Account
 
+from martins_script import Order as OldOrder
+from order import Order
 
-class OrderTests(unittest.TestCase):
+
+class OldOrderTests(unittest.TestCase):
     def test_opposite_kind(self):
-        buy_order = Order(sell_token="TokenA", buy_token="TokenB", receiver="",
-                          sell_amount=1, buy_amount=2, valid_to=0, fee_amount=1,
-                          kind="buy")
+        buy_order = OldOrder(sell_token="TokenA", buy_token="TokenB", receiver="",
+                             sell_amount=1, buy_amount=2, valid_to=0, fee_amount=1,
+                             kind="buy")
         self.assertEqual(buy_order.opposite_kind(), "sell")
-        sell_order = Order(sell_token="TokenA", buy_token="TokenB", receiver="",
-                           sell_amount=1, buy_amount=2, valid_to=0, fee_amount=1,
-                           kind="sell")
+        sell_order = OldOrder(sell_token="TokenA", buy_token="TokenB", receiver="",
+                              sell_amount=1, buy_amount=2, valid_to=0, fee_amount=1,
+                              kind="sell")
         self.assertEqual(sell_order.opposite_kind(), "buy")
 
 
+class OrderTests(unittest.TestCase):
+    maxDiff = None
```

Comment:
Is this used?

Reply:
Only in the testing to get full results instead of partial with ... between beginning and end